docs/content/concepts/metadata-tags/asset-metadata/column-level-lineage.mdx

```diff
@@ -0,0 +1,146 @@
+                        TableColumnDep(
+                            asset_key=AssetKey("source_bar"),
+                            column_name="column_quuz",
+                        ),
+                    ],
+                }
+            )
+        }
+    )
+```
+
+When materialized, the `my_asset` asset will create two columns: `new_column_foo` and `new_column_qux`.
+
+The `new_column_foo` column is dependent on two other columns:
+
+1. `column_bar` from the `source_bar` asset
+2. `column_baz` from the `source_baz` asset
+
+And the second column, `new_column_qux` has is dependent on `column_quuz` from the `source_bar` asset.
+
+Now, you can view the lineage for these columns in the [Dagster UI](#viewing-column-lineage-in-the-dagster-ui).
+
+### For assets loaded from integrations
+
+Column-level lineage is currently supported for the dbt integration. Refer to the [dbt documentation](/integrations/dbt/reference/#emit-column-level-metadata-as-materialization-metadata) for more information.
+
+---
+
+## Viewing column lineage in the Dagster UI
+
+1. In the Dagster UI, open the **Asset details** page for an asset with column-level lineage enabled.
+2. Navigate to the **Overview** tab if it isn't already open.
+3. In the **Columns** section, click the **branch** icon in the row of the column you want to view. The icon is on the far right side of the row:
+
+   <!-- ![Highlighted column lineage icon in the Asset details page of the Dagster UI](/images/concepts/metadata-tags/column-lineage-icon.png) -->
+
+   <Image
+   alt="Highlighted column lineage icon in the Asset details page of the Dagster UI"
+   src="/images/concepts/metadata-tags/column-lineage-icon.png"
+   width={905}
+   height={193}
+   />
+
+The graph will display the column's asset and column dependencies:
+
+<!-- ![Column lineage for a credit_utilization column in the Dagster UI](/images/concepts/metadata-tags/column-level-lineage.png) -->
+
+<Image
+alt="Column lineage for a credit_utilization column in the Dagster UI"
+src="/images/concepts/metadata-tags/column-level-lineage.png"
+width={2640}
+height={1276}
+/>
+
+To view another column's lineage, click the **Column** dropdown and select another column.
+
+---
+
+## APIs in this guide
+
+| Name                                     | Description                                                             |
+| ---------------------------------------- | ----------------------------------------------------------------------- |
+| <PyObject object="asset" decorator />    | A decorator used to define assets.                                      |
+| <PyObject object="MaterializeResult" />  | An object representing a successful materialization of an asset.        |
+| <PyObject object="TableColumnLineage" /> | Class that defines column outputs to column inputs for a tabular asset. |
+| <PyObject object="TableColumnDep" />     | Object that represents a column in an asset.                            |
+
```

Comment:
Could this be higher up? I think "Relevant APIs" should be before "How it works", similar to our other concepts.

Reply:
I'm actually in the process of moving these down the page for all our concept pages. Some of the pages have tables that are *extremely* long and it requires a lot of scrolling to get to the actual content - I've heard a good amount of feedback that it's fairly disliked.